client/src/pages/PackageDetailsScreen/FlightCard/FlightDetails.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Typography, Button } from 'antd';
+import { ArrowRightOutlined } from '@ant-design/icons';
+import styles from './scss/flight-details.module.scss';
+import { formattedDate } from '@/utils/date.utils';
+
+const { Text } = Typography;
+
+interface FlightDetailsProps {
+    from: string;
+    to: string;
+    departureDate: string;
+    price: number;
+    linkForTicket: string;
+}
+
+const FlightDetails = ({ from, to, departureDate, price,linkForTicket }: FlightDetailsProps) => {
+    return (
+        <div className={styles['flight-details-container']}>
```

Comment:
```suggestion
        <div className={styles.flightDetailsContainer}>
```
I whink this notation is looking nicer but not critical. I know that if you use jetbrains webstorm or vscode css module extension it would autocomplete you to those

Reply:
Look for others as well